docs/guide/docs/managing/features.md

```diff
@@ -0,0 +1,115 @@
+---
+id: features
+title: Useful Features
+---
+
+## Shortlinks
+
+In Botpress, you can natively create short links to your bot and get the following benefits:
+
+1. Short URLs - no one likes a long URL
+2. Flexibility - it allows you to change any of the parameters without affecting the URL
+
+### Implementation
+In the example below, our new shortlink `/s/fs-wc` will redirect a user to `/lite/botId?m=platform-webchat&v=fullscreen` (the standard webchat interface). You can specify additional parameters in the options object.
+
+Create a bot-scoped `after_bot_mount` hook with the following code:
+
+```js
+bp.http.createShortLink('fs-wc', `${process.EXTERNAL_URL}/lite/${botId}/`, {
+  m: 'channel-web',
+  v: 'fullscreen',
+  options: JSON.stringify({
+    config: {
+      /* Custom config here... */
+    }
+  })
+})
+```
+### Resources
+See the views' [Config](https://github.com/botpress/botpress/blob/master/modules/channel-web/src/views/lite/typings.d.ts#L130) object for all available options.
+
+It is recommended to also create a hook `after_bot_unmount`, to remove the shortlink when the bot is disabled; here is the corresponding example:
+
+```js
+bp.http.deleteShortLink('fs-wc')
+```
+## Listening For File Changes
+
+You may find yourself writing custom logic when a Botpress file has changed. For example, you could listen for changes to the QnA files to automatically launch a translation worker to translate the QnA to multiple languages.
+
+The Botpress File System (Ghost) exposes a way to listen for file changes for that purpose. In this example, we will watch for NLU changes inside any bot.
+
+### Example
+
+Let's create a Hook inside the `<data_dir>/global/hooks/after_bot_mount` called `listen_nlu.js` and put the following code inside it:
+
+```js
+const listener = bp.ghost.forBot(botId).onFileChanged(file => {
+  if (
+    file.toLowerCase().startsWith(`data/bots/${botId}/intents/`) ||
+    file.toLowerCase().startsWith(`data/bots/${botId}/entities/`)
+  ) {
+    bp.logger.info('NLU Data has changed: ' + file)
+  }
+})
+
+setTimeout(() => {
+  // Example of how to stop listening after 1m
+  listener.remove()
+}, 60 * 1000)
+```
+
+## Inter-bot Communication / Delegation
+
+Sometimes your chatbot needs to "delegate" questions or tasks to other bots. We call this concept "inter-bot" communication.
+
+The code in this example is available in the [examples](https://github.com/botpress/botpress/tree/master/examples/interbot) directory of our GitHub repository (update `workspaces.json` with the three bots if you copied them).
+
+![Example](assets/tutorials_interbot-example.png)
+
+### Structure
+
+![Diagram](assets/tutorials_interbot-diagram.png)
+
+### Step 1 – Creating the bots

```

Comment:
pretty sure @marcmerce would want chatbots here and and everywhere else

Reply:
Okay reviewing...